css/chat.css

```diff
@@ -75,7 +75,7 @@ html.no-supports-container.native body,
 .chat-holder {
   display: flex;
   position: absolute;
-  top: 0;
+  top: -40px;
```

Comment:
[@squallstar's concern](https://github.com/Fliplet/fliplet-widget-chat/pull/106#issuecomment-726122018) is valid.

`body` has the classes `fl-with-top-menu` or `fl-with-bottom-menu` depending on whether there's a menu at the top or bottom. You can make use of this information if you want to assess the presence of top and bottom menus.

In addition, avoid using specific pixel numbers that rely on the design/appearance of other components. `position: fixed`, for example, would be a pretty straightforward method to anchor something to the viewport using CSS.